tools/compile-proto-file/Main.hs

```diff
@@ -47,5 +47,10 @@ parseArgs = info (helper <*> parser) (fullDesc <> progDesc "Compiles a .proto fi
       $ long "largeRecords"
       <> help "Use large-records library to optimize the core code size of generated records"
 
+    isPrefixed = IsPrefixed . not <$> switch (
+      long "no-field-prefix"
+      <> help "Remove prefix for record field names"
```

Comment:
This is a nitpicking, but perhaps more natural explanation of the behaviour would be "do not prefix record field names by the type names", since it does not remove anything when this is enabled.